packages/flutter/test/widgets/scrollbar_test.dart

```diff
@@ -1374,6 +1374,37 @@ void main() {
     );
   });
 
+  testWidgets('The bar can show or hide when the viewport size change', (WidgetTester tester) async {
```

Comment:
Can you add another test like this, where rather than changing the content (inside of SingleChildScrollView), it changes the size of the window around it just to verify it works both ways? I think you can do this easiest by just wrapping it with a media query and altering the media query data.

Reply:
OK, I will add more tests for the new events.

Reply:
Is this change still break the G-test or internal test?

Reply:
I'll run the test suite with the latest commit. It only runs automatically after something has been approved.